setup.sh

```diff
@@ -26,6 +26,7 @@ fi
 rustup update nightly
 rustup target add wasm32-unknown-unknown --toolchain nightly
 rustup update stable
+rustup default nightly
```

Comment:
Is nightly necessary for the current release?

Reply:
I can remove, may have added that accidentally